stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -302,6 +269,11 @@ extension DoubleWidth : FixedWidthInteger {
     return High.bitWidth + Low.bitWidth
   }
 
+  @_inlineable // FIXME(sil-serialize-all)
+  public var bitWidth: Int {
```

Comment:
Is this necessary? I thought the default `FixedWidthInteger` implementation would take care of it.

Reply:
The compiler had trouble recognizing that there's a default implementation. The redundant definitions for indexing operations below are there for the same reason. Some of these can probably be eliminated by careful reorganization, but I suspect there might be an underlying compiler issue that needs to be investigated.

Reply:
That’s rather unfortunate. I wonder what change brought that about given that it’s fine at the moment.

Reply:
It's triggered by adding the `RandomAccessCollection` requirement; I think `Collection` causes similar issues.

Reply:
I can imagine how that might arise for the indexing operations, but for `bitWidth`?